torchtext/models/roberta/bundler.py

```diff
@@ -156,10 +155,13 @@ def encoderConf(self) -> RobertaEncoderConf:
 XLMR_BASE_ENCODER = RobertaModelBundle(
     _path=urljoin(_TEXT_BUCKET, "xlmr.base.encoder.pt"),
     _encoder_conf=RobertaEncoderConf(vocab_size=250002),
-    transform=partial(get_xlmr_transform,
-                      vocab_path=urljoin(_TEXT_BUCKET, "xlmr.vocab.pt"),
-                      spm_model_path=urljoin(_TEXT_BUCKET, "xlmr.sentencepiece.bpe.model"),
-                      )
+    transform=lambda: T.Sequential(
+        T.SentencePieceTokenizer(urljoin(_TEXT_BUCKET, "xlmr.sentencepiece.bpe.model")),
+        T.VocabTransform(load_state_dict_from_url(urljoin(_TEXT_BUCKET, "xlmr.vocab.pt"))),
+        T.Truncate(510),
```

Comment:
I am curious as to why we are truncating to a length of 510 exactly? In the XLMR transform that is being removed, I noticed that `max_seq_len` was an input arg with a default value of 514 and we would truncate to a length of `self.max_seq_len - 2` which would be 512 instead of 510 right?

Reply:
I think 510 + the BOS and EOS is right. The 514 seems like an off-by-two.

Reply:
Sounds like some tests would help.